templates/simple/environments/_base/www.ts

```diff
@@ -0,0 +1,29 @@
+import { AppComponentEnvironment } from "@socialgouv/kosko-charts/components/app/params";
+import { ok } from "assert";
+
+ok(process.env.CI_REGISTRY_IMAGE);
+ok(process.env.CI_COMMIT_SHA);
+
+const env: AppComponentEnvironment = {
+  containerPort: 8080,
+
+  image: {
+    name: process.env.CI_REGISTRY_IMAGE,
+    tag: process.env.CI_COMMIT_TAG
+      ? process.env.CI_COMMIT_TAG.slice(1)
+      : process.env.CI_COMMIT_SHA,
+  },
+
+  ingress: {
+    secretName: process.env.PRODUCTION ? "www-crt" : "wildcard-crt",
+  },
+
+  labels: {
+    component: "next",
```

Comment:
`CI_PROJECT_NAME` ?

Reply:
Here `component` refers to [`app.kubernetes.io/component`](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels) which is a concrete description of what is running in the Pod.